test/unit/storage_async_test.rb

```diff
@@ -247,8 +250,8 @@ def test_tls_no_client_certificate
           key: key.path
         }
       }
-      storage = StorageAsync::Client.send :new, Storage::Helpers.config_with(config_obj)
-      assert_client_config(config_obj, storage, alg)
+      storage = assert_nothing_raised { StorageAsync::Client.send :new, Storage::Helpers.config_with(config_obj) }
```

Comment:
why do you add these? I think we once discussed that in a test everything is assumed not to raise. And this assertion only messes up the backtrace.